packages/runner/src/main.js

```diff
@@ -70,15 +69,6 @@ function createWindow () {
     mainWindow.webContents.openDevTools({ mode: 'detach' })
   }
 
-  ipcMain.on('study.load', (e, filePaths) => {
-    console.log('loading file paths from', filePaths)
-    // const studyWindow = new StudyWindow(filePaths, {
-    //   development: inDevelopment,
-    // })
-    //const window = remote.BrowserWindow;
-
-  })
-
```

Comment:
Ok, this is shaping up nicely! Out of interest, why is this block missing?

Reply:
It is just a WIP Code. I removed it because it does not have any use here. I am developing on this and would be using it in my next contributions.